src/content/components/BugList/BugList.scss

```diff
@@ -210,19 +210,7 @@ $monospace: SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New"
   }
 }
 
-.task, .defect, .enhancement {
+.bugIcon {
   vertical-align: top;
   margin-top: -2px;
```

Comment:
I wonder if we still need these rules now. Never looked too closely at this, but at a glance it seems like these might have been intended to compensate for the typesetting weirdness. Can you look into that?

Reply:
i.e. does removing these affect the vertical centering of the icons relative to the text? we might still want to keep them but worth checking